src/main/java/edu/miu/cs/cs544/EAProject/domain/StateProvince.java

```diff
@@ -10,7 +11,7 @@
 @Entity
 @Table(name = "StateProvince")
 @NoArgsConstructor
-@Getter
```

Comment:
I just wanna allow some properties to change the value in runtime. That's why I didn't use @Data